### DISCUSSION TITLE
HTML5 Glitch im having

### DISCUSSION THREAD

**Discussion by yeaaaaaahhhhh** [FunkinCrew/Funkin#1925] [Compiling help]
Project.xml?????!?!?!? i have it in the files! but it doesn't work
![Command Prompt 1_6_2022 10_11_36 AM](https://user-images.githubusercontent.com/75813332/148413580-1dea1ccc-e5e7-4036-99b2-73b9a101e415.png)

**Comment by whywhywhyw** [comment]
you need to cd into the directory
then run lime

**Comment by NokioPyro** [comment]
Your in the wrong directory lmao, When you open cmd / powershell dont have a folder selected.

**Comment by libauror-a** [comment]
either navigate to the directory, or specify the project.xml file in the folder path
